### PR TITLE
PR: BAEL-8254

### PR DESCRIPTION
**Pull request by MohamedHelmyKassab**
This PR aims to add files and test class regarding BAEL-8254

### CODE REVIEW

File: core-java-modules/core-java-io-7/pom.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>core-java-io-7</artifactId>
+    <packaging>jar</packaging>
+    <name>core-java-io-7</name>
```
Comment (KevinGilmore) [inline core-java-modules/core-java-io-7/pom.xml:8]: Please swap these two lines (as per guidelines).

Comment (Bwvolleyball) [inline core-java-modules/core-java-io-7/pom.xml:8]:
```suggestion
    <name>core-java-io-7</name>
    <packaging>jar</packaging>

```

**Comment by Bwvolleyball** [comment]
retest this please

**Comment by Bwvolleyball** [comment]
retest this please

**Comment by Bwvolleyball** [comment]
retest this please

**Comment by Bwvolleyball** [comment]
retest this please

**Pull request merged**